pkg/intoto/slsa_provenance/v0.2/provenance.go

```diff
@@ -185,3 +186,22 @@ func ParseSLSAv02Predicate(predicate interface{}) (*ProvenancePredicate, error)
 
 	return &pp, nil
 }
+
+// GetConfigSourceURI returns the URI of the configuration.
+func GetConfigSourceURI(pred ProvenancePredicate) string {
```

Comment:
Having looked at the provenance examples again, I'd suggest removing this entirely as it is either not used (in `amber`) or is redundant (in `slsav02`).